lib/command/info.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Command
+  class Info < Base
+    NAME = "info"
+    OPTIONS = [
+      org_option,
+      app_option
+    ].freeze
+    DESCRIPTION = "Displays a list of available workloads for all GVCs or a specific GVC in an org"
```

Comment:
We need to resolve the calling of GVC vs. app. Below, we have $APP_NAME.

For the info, we should be displaying Apps

```
DESCRIPTION = "Displays a list of available workloads for all Apps or a specific App in an org. Apps equal GVCs"
```